### PR TITLE
Blog post for v1.0.0 release

### PR DESCRIPTION
**Pull request by 6543**
introduce the v1.0.0

blocked till tag got released

### CODE REVIEW

File: docs/blog/2023-07-28-release-v1.0.0/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - name: 6543
+    title: Maintainer of Woodpecker
+    url: https://github.com/6543
+    image_url: https://github.com/6543.png
+tags: [release, stable]
+hide_table_of_contents: false
+---
+
+We are proud to present you Woodpecker v1.0.0.
+It took us quite some time, but now we are sure it's ready, and so you should really have a look at it.
```
Comment (xoxys) [inline docs/blog/2023-07-28-release-v1.0.0/index.md:15]:
```suggestion
It took us quite some time, but now we are sure it's ready, and you should really have a look at it.
```

File: docs/blog/2023-07-28-release-v1.0.0/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+<!--truncate-->
+
+We've refactored a lot of code, so contributing to the codebase should be much easier.
```
Comment (xoxys) [inline docs/blog/2023-07-28-release-v1.0.0/index.md:19]:
```suggestion
We've refactored a lot of code, contributing to the codebase should be much easier.  
```

```suggestion
We've refactored a lot of code and contributing should be much easier now.  
```

File: docs/blog/2023-07-28-release-v1.0.0/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Automate recurring tasks with ease using Woodpecker's new cron jobs feature.
+Schedule pipelines to run at specified intervals or times, optimizing repetitive workflows.
+[Read more](/docs/usage/cron)
+
+### YAML Map Merge, Overrides, and Sequence Merge Support
+
+With enhanced YAML support, managing complex configurations becomes a breeze.  
+Merge maps, apply overrides, and sequence merging—all within your YAML files.
+This is providing greater flexibility and control over your pipelines.
```
Comment (xoxys) [inline docs/blog/2023-07-28-release-v1.0.0/index.md:36]:
```suggestion
This is providing more flexibility and control over your pipelines.
```

File: docs/blog/2023-07-28-release-v1.0.0/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+### Localize Web-UI
+
+Embrace internationalization by changing your locale in the user settings.  
+Interact with Woodpecker in the language of your choice, tailored to your preferences.  
+If your language is not available or only partially translated, consider contributing to our [Weblate](https://translate.woodpecker-ci.org/engage/woodpecker-ci/).
+
+### Add `evaluate` to `when` Filter
+
+Enhance pipeline flexibility with the new "when evaluate" filter, enabling or disabling steps based on custom conditions.
+Tailor your workflows to respond dynamically to specific triggers and events.
```
Comment (xoxys) [inline docs/blog/2023-07-28-release-v1.0.0/index.md:55]:
```suggestion
Customize your workflows to dynamically respond to specific triggers and events.
```

**Review by xoxys — APPROVED** [review]
I like it but you might want to wait for other opinions.

File: docs/blog/2023-07-28-release-v1.0.0/index.md (hunk 2 above)
Comment (xoxys) [inline docs/blog/2023-07-28-release-v1.0.0/index.md:19]:
```suggestion
We've refactored a lot of code and contributing to the codebase should be much easier now.
```

File: docs/blog/2023-07-28-release-v1.0.0/index.md (hunk 4 above)
Comment (6543) [reply docs/blog/2023-07-28-release-v1.0.0/index.md:55]: the `  ` at the end is intended to enforce a newline

File: docs/blog/2023-07-28-release-v1.0.0/index.md (hunk 2 above)
Comment (6543) [reply docs/blog/2023-07-28-release-v1.0.0/index.md:19]: well it could be better worded but I personally prefer the current wording over the suggested one

Comment (xoxys) [reply docs/blog/2023-07-28-release-v1.0.0/index.md:19]: Thats totally fine

**Comment by 6543** [comment]
note for us: comment on https://github.com/mahmoud/zerover/issues/135 once this is out :D

**Pull request merged**